airflow/models/__init__.py

```diff
@@ -2999,15 +2999,29 @@ class DagModel(Base):
     fileloc = Column(String(2000))
     # String representing the owners
     owners = Column(String(2000))
+    # Description of the dag
+    description = Column(Text)
+    # Default view of the inside the webserver
+    default_view = Column(String(25))
 
     def __repr__(self):
         return "<DAG: {self.dag_id}>".format(self=self)
 
+    @property
+    def timezone(self):
+        return settings.TIMEZONE
```

Comment:
Wait up - this looks incorrect. Why was this removed?

The timezone of a DAG was previously defined as the timezone of the start_date. Is that still the case and if we do `self.timezone = tz` then this property accessor won't be used anymore?

Reply:
I don't  mean removed, I mean added.

Reply:
Oh this is on the DagModel, not the DAG. Right.

In which case my question is: when/where is this used?

Reply:
This is indeed a bit tricky. I have added this because in the html templates this variable is used. Some templates now takes a DagModel instead of a DAG. In the end I would like this to be in the complete webserver